content/blog/vs_code_extension_blog.md

```diff
@@ -0,0 +1,52 @@
+﻿---
+
+title: Learning with Practise (My Experience at FOSS Overflow)
+
+author: Aayush Deshmukh
+
+date: '2022-01-21'
+
+---
+
+# Learning With Practise
+
+Learning new things with hands-on experience is an excellent way of acquiring a new skill. It is not only a faster and effective method but also you get to tackle errors which you will face in real life projects, so you will be ready to deal with those.
+
+*"Through practice, gently and gradually we can collect ourselves and learn how to be more fully with what we do. – Jack Kornfield"*
+
+ ---
+  
+[**FOSS Overflow**](https://fossoverflow.dev/) is a month-long event celebrating open source culture and is organized by Google Developer Student Clubs and OpenLake at **IIT Bhilai**. The program is designed to prepare students for global open source internships like GSoC, LFX, and Outreachy. Foss Overflow gave me a great opportunity to learn something new while working on a project. 
+
+<div align="center">
+
+  
+
+![Alt text](https://fossoverflow.dev/nextimg/%2F_next%2Fstatic%2Fmedia%2Flogo.4bd48c29.png/828/75?url=%2F_next%2Fstatic%2Fmedia%2Flogo.4bd48c29.png&w=828&q=75)
```

Comment:
This image URL may break. Refer https://github.com/OpenLake/FOSS-Overflow/pull/2/files for including images

Reply:
added the pic in the repo.